scss/_responsive.scss

```diff
@@ -30,4 +30,25 @@
     margin: -$em;
     padding: $em;
   }
+  
+  textarea,
+  input,
+  select {
+    max-width: 100%;
+  }
+  
+  fieldset {
+    min-width: 0;
+  }
+  
+  // Firefox bug fix
```

Comment:
would be nice to add a link here to some web page explaining this bug. now you know what this bug is about, but tomorrow someone else will have hard time understanding what exactly did you mean here